scss/utilities/_spacing.scss

```diff
@@ -8,34 +8,42 @@
       @each $size, $length in $spacers {
 
         .#{$abbrev}#{$infix}-#{$size}  { #{$prop}:        $length !important; }
-        .#{$abbrev}t#{$infix}-#{$size} { #{$prop}-top:    $length !important; }
-        .#{$abbrev}r#{$infix}-#{$size} { #{$prop}-right:  $length !important; }
-        .#{$abbrev}b#{$infix}-#{$size} { #{$prop}-bottom: $length !important; }
-        .#{$abbrev}l#{$infix}-#{$size} { #{$prop}-left:   $length !important; }
+        .#{$abbrev}t#{$infix}-#{$size},
+        .#{$abbrev}y#{$infix}-#{$size} {
+          #{$prop}-top:    $length !important;
+        }
+        .#{$abbrev}r#{$infix}-#{$size},
         .#{$abbrev}x#{$infix}-#{$size} {
-          #{$prop}-right: $length !important;
-          #{$prop}-left:  $length !important;
+          #{$prop}-right:  $length !important;
         }
+        .#{$abbrev}b#{$infix}-#{$size},
         .#{$abbrev}y#{$infix}-#{$size} {
-          #{$prop}-top:    $length !important;
           #{$prop}-bottom: $length !important;
         }
+        .#{$abbrev}l#{$infix}-#{$size},
+        .#{$abbrev}x#{$infix}-#{$size} {
+          #{$prop}-left:   $length !important;
+        }
       }
     }
 
     // Some special margin utils
     .m#{$infix}-auto  { margin:        auto !important; }
-    .mt#{$infix}-auto { margin-top:    auto !important; }
-    .mr#{$infix}-auto { margin-right:  auto !important; }
-    .mb#{$infix}-auto { margin-bottom: auto !important; }
-    .ml#{$infix}-auto { margin-left:   auto !important; }
+    .mt#{$infix}-auto,
+    .my#{$infix}-auto {
+      margin-top:    auto !important;
+    }
+    .mr#{$infix}-auto,
     .mx#{$infix}-auto {
-      margin-right: auto !important;
-      margin-left:  auto !important;
+      margin-right:  auto !important;
```

Comment:
Can you remove the spaces after `:` that don't make sense to have like here and the other places you edited?